### ISSUE TITLE
Invalid Bundle Structure

### ISSUE THREAD

**Issue by paulflores** [inthekloud/ITK.Instabug#2]
![screen shot 2015-02-04 at 4 06 49 pm](https://cloud.githubusercontent.com/assets/1565635/6052480/2aeb9de0-ac88-11e4-9d23-7e37918296de.png)

I'm getting this error when trying to validate my app archive.

A quick google search of the error suggests that an executable file was added to the project.

Also any plans on wrapping the android sdk of instabug as a module?

Awesome work
Thanks
Paul

**Comment by AhmedTheGeek** [comment]
Yep, i ran through that error while packaging my own app, working on a fix for that, and yeah sure, working on the android version, too ;) Will update you on this issue..

**Comment by iRonin** [comment]
+1 Can't build our app for inflight testing :(

**Comment by AhmedTheGeek** [comment]
Will upload fixed version today.. sorry about that...

**Comment by iRonin** [comment]
What needs to be done to fix this?

**Comment by AhmedTheGeek** [comment]
Try downloading Version 1.1.1 from dist folder, it should fix that issue. Try it an tell me if doesn't fix it for you.

**Comment by paulflores** [comment]
Still does not work, the new version won't compile with titanium now.

**Comment by iRonin** [comment]
Does not work for me either:

```
[TRACE] ld: warning: directory not found for option '-F/Users/iRonin/Work/myapp/repos/myapp/build/iphone/../../modules/iphone/itk.instabug/1.1.1/platform/iphone'
[TRACE] ld: framework not found Instabug
[TRACE] clang: error: linker command failed with exit code 1 (use -v to see invocation)
[ERROR] ** BUILD FAILED **
[ERROR] The following build commands failed:
[ERROR]         Ld build/myapp.build/Release-iphoneos/myapp-universal.build/Objects-normal/armv7/myapp normal armv7
[ERROR]         Ld build/myapp.build/Release-iphoneos/myapp-universal.build/Objects-normal/arm64/myapp normal arm64
[ERROR] (2 failures)
```

**Comment by SteffanPB** [comment]
Still getting the following error:

![screen shot 2015-03-31 at 16 32 51](https://cloud.githubusercontent.com/assets/6866035/6921252/e40d8dfe-d7c3-11e4-921c-b15fd82d22a6.png)

**Comment by caspahouzer** [comment]
Any news on that?
I really want to use this awesome module in production

**Comment by AhmedTheGeek** [comment]
@caspahouzer i'll fiddle around with it again tonight, and will keep you updated. ;)

**Comment by caspahouzer** [comment]
Thank you very much!

- Sebastian

**Comment by iRonin** [comment]
:+1:

**Comment by AhmedTheGeek** [comment]
I have a good news, and bad news.. the good news that i fixed the invalid bundle structure issue. the bad news that now the instabug framework can't find the bundle assets, like images and locales. so i'm trying to find solution for that. just wanted to keep you updated.

**Comment by caspahouzer** [comment]
👍🏻

- Sebastian 

Sent from my iPhone

**Comment by pravin-mindstix** [comment]
Facing Same issue, And really need a urgent help.

Is there any solution. I am using instabug 1.1.1.

Thanks,
